hack/generate.sh

```diff
@@ -50,7 +50,7 @@ fi
 
 client-gen --clientset-name versioned \
     --input-base kubevirt.io/api \
-    --input export/v1alpha1,snapshot/v1alpha1,instancetype/v1alpha1,instancetype/v1alpha2,instancetype/v1beta1,pool/v1alpha1,migrations/v1alpha1,clone/v1alpha1 \
+    --input core/v1,export/v1alpha1,snapshot/v1alpha1,instancetype/v1alpha1,instancetype/v1alpha2,instancetype/v1beta1,pool/v1alpha1,migrations/v1alpha1,clone/v1alpha1 \
```

Comment:
Why mixing this v1 change with the shadow node changes? It is worth to be a separate CL in my opinion.

Reply:
I am using the generated client. What do you mean by CL?

Reply:
CL means PR in this context.